src/main/java/com/xiaomi/infra/pegasus/rpc/ThriftHeader.java

```diff
@@ -6,32 +6,18 @@
 import java.nio.ByteBuffer;
 
 public class ThriftHeader {
-  public static final int HEADER_LENGTH = 48;
+  public static final int HEADER_LENGTH = 16;
```

Comment:
Will update `HEADER_LENGTH` directly leading to incompatibility?

Reply:
What does imcompatibility mean? 
The new version client can access old version server. I have already tested it.

Reply:
Okay~ Your comment has already answered my question~ I have another one. What about the old version client try to access the new version server?

Reply:
Sorry I have made a mistake.
old/new client --> new client is ok.
old client --> old client is ok, too.
but new client --> old server is not ok, because the connection will be disconnected by server. So we can`t get any information about version imcompatibility, It simply disconnects the connection.